<style>
<!--
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	font-size:10.5pt;
	font-family:"Times New Roman";
	margin-left:0cm; margin-right:0cm; margin-top:0cm}
-->
</style>

<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
</head>


<div>
	<table border="0" id="table1" bgcolor="#FFFFFF">
<tr>
	<td valign="baseline" colspan="3" bgcolor="#ADD8E6">
		<p><b>
			<font face="Times" size="3">
				<a name="2023">Protein-DNA Binding Site Predition</a>
			</font>
		</b></p>
	</td>
</tr>
		<tr>
		<td valign="baseline" colspan="3">
		<ul>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			中文综述：识别蛋白质配体绑定残基的生物计算方法综述.
			<b> 数据采集与处理</b>, 2018.
			[<a href="./papers/2/Chinese Overview.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Advancing protein-DNA binding site prediction: integrating sequence models and machine learning classifiers.
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/2/English Overview 1.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NABind: Structure-based prediction of nucleic acid binding residues by merging deep learning and template-based approaches.
			<b> PLoS Computational Biology</b>, 2023.
			[<a href="./papers/2/NABind_PLOSCB_2023.pdf">PDF</a>]
			[<a href="http://liulab.hzau.edu.cn/NABind/" target=_blank>Web Server</a>]
			[<a href="https://github.com/hzau-liulab/NABind" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ULDNA: Integrating unsupervised multi-source language models with LSTM-attention network for protein-DNA binding site prediction.
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/2/ULDNA_bioRxiv_2023.pdf">PDF</a>]
			[<a href="http://csbio.njust.edu.cn/bioinf/uldna/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CLAPE: Protein-DNA binding sites prediction based on pre-trained protein language model and contrastive learning.
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/2/CLAPE_bioRxiv_2023.pdf">PDF</a>]
			[<a href="https://github.com/YAndrewL/clape/" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GLMSite: Accurately identifying nucleic-acid-binding sites through geometric graph learning on language model predicted structures.
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/2/GLMSite_bioRxiv_2023.pdf">PDF</a>]
			[<a href="https://github.com/biomed-AI/nucleic-acid-binding" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			EquiPNAS: Improved protein-nucleic acid binding site prediction using protein-language-model-informed equivariant deep graph neural networks.
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/2/EquiPNAS_bioRxiv_2023.pdf">PDF</a>]
			[<a href="https://github.com/Bhattacharya-Lab/EquiPNAS" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PredDBR: Protein-DNA binding residue prediction via bagging strategy and sequence-based cube-format feature.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2022.
			[<a href="./papers/2/PredDBR_TCBB_2022.pdf">PDF</a>]
			[<a href="https://jun-csbio.github.io/PredDBR/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DBpred: A deep learning-based method for the prediction of DNA interacting residues in a protein.
			<b> Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/2/DBPred_Briefings in Bioinformatics_2022.pdf">PDF</a>]
			[<a href="https://webs.iiitd.edu.in/raghava/dbpred/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Guan's Method: Protein-DNA binding residues prediction using a deep learning model with hierarchical feature extraction.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2022.
			[<a href="./papers/2/Guan's Method_TCBB_2022.pdf">PDF</a>]
			[<a href="https://github.com/ShixuanGG/DNA-protein_binding_residues" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GraphSite: AlphaFold2-aware protein-DNA binding site prediction using graph transformer.
			<b> Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/2/GraphSite_BIB_2022.pdf">PDF</a>]
			[<a href="https://biomed.nscc-gz.cn/apps/GraphSite" target=_blank>Web Server</a>]
			[<a href="https://github.com/biomed-AI/GraphSite" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepDISOBind: Accurate prediction of RNA-, DNA- and protein-binding intrinsically disordered residues with deep multi-task learning.
			<b> Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/2/DeepDISOBind_BIB_2022.pdf">PDF</a>]
			[<a href="https://www.csuligroup.com/DeepDISOBind/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			BindWeb: A web server for ligand binding residue and pocket prediction from protein structures.
			<b> Protein Science</b>, 2022.
			[<a href="./papers/2/BindWeb_Protein Science_2022.pdf">PDF</a>]
			[<a href="http://www.csbio.sjtu.edu.cn/bioinf/BindWeb/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			iDRNA-ITF: Identifying DNA- and RNA-binding residues in proteins based on induction and transfer framework.
			<b> Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/2/iDRNA-ITF_BIB_2022.pdf">PDF</a>]
			[<a href="http://bliulab.net/iDRNA-ITF/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NCBRPred: Predicting nucleic acid binding residues in proteins based on multilabel learning.
			<b> Briefings in Bioinformatics</b>, 2021.
			[<a href="./papers/2/NCBRPred_BIB_2021.pdf">PDF</a>]
			[<a href="http://bliulab.net/NCBRPred" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GraphBind: Protein structural context embedded rules learned by hierarchical graph neural networks for recognizing nucleic-acid-binding residues.
			<b> Nucleic Acids Research</b>, 2021.
			[<a href="./papers/2/GraphBind_NAR_2021.pdf">PDF</a>]
			[<a href="http://www.csbio.sjtu.edu.cn/bioinf/GraphBind/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNAgenie: Accurate prediction of DNA-type-specific binding residues in protein sequences.
			<b> Briefings in Bioinformatics</b>, 2021.
			[<a href="./papers/2/DNAgenie_BIB_2021.pdf">PDF</a>]
			[<a href="http://biomine.cs.vcu.edu/servers/DNAgenie/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProNA2020 predicts protein-DNA, protein-RNA, and protein-protein binding proteins and residues from sequence.
			<b> Journal of Molecular Biology</b>, 2020.
			[<a href="./papers/2/ProNA2020_JMB_2020.pdf">PDF</a>]
			[<a href="https://predictprotein.org/" target=_blank>Web Server</a>]
			[<a href="https://github.com/Rostlab/ProNA2020" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNAPred: Accurate identification of DNA-binding sites from protein sequence by ensembled hyperplane-distance-based support vector machines.
			<b> Journal of Chemical Information and Modeling</b>, 2019.
			[<a href="./papers/2/DNAPred_JCIM_2019.pdf">PDF</a>]
			[<a href="http://csbio.njust.edu.cn/bioinf/dnapred/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NUCBind: Improving the prediction of protein-nucleic acids binding residues via multiple sequence profiles and the consensus of complementary methods.
			<b> Bioinformatics</b>, 2019.
			[<a href="./papers/2/NUCBind(COACH-D+SVMnuc)_Bioinformatics_2019.pdf">PDF</a>]
			[<a href="https://yanglab.nankai.edu.cn/NucBind/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			iProDNA-CapsNet: Identifying protein-DNA binding residues using capsule neural networks.
			<b> BMC Bioinformatics</b>, 2019.
			[<a href="./papers/2/iProDNA-CapsNet_BMC Bioinformatic_2019.pdf">PDF</a>]
			[<a href="https://github.com/ngphubinh/iProDNA-CapsNet" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HybridNAP: Comprehensive review and empirical analysis of hallmarks of DNA-, RNA- and protein-binding residues in protein chains.
			<b> Briefings in Bioinformatics</b>, 2019.
			[<a href="./papers/2/HybridNAP_BIB_2017.pdf">PDF</a>]
			[<a href="http://biomine.cs.vcu.edu/servers/hybridNAP/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			funDNApred: Prediction of DNA-binding residues in local segments of protein sequences with fuzzy cognitive maps.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2018.
			[<a href="./papers/2/funDNApred_TCBB_2018.pdf">PDF</a>]
			[<a href="http://biomine.cs.vcu.edu/servers/funDNApred/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TargetDNA: Predicting protein-DNA binding residues by weightedly combining sequence-based features and boosting multiple SVMs.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2016.
			[<a href="./papers/2/TargetDNA_TCBB_2016.pdf">PDF</a>]
			[<a href="http://csbio.njust.edu.cn/bioinf/targetdna/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CNNsite: Prediction of DNA-binding residues in proteins using convolutional neural network with sequence features.
			<b> IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</b>, 2016.
			[<a href="./papers/2/CNNsite_BIBM_2016.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PDNAsite: Identification of DNA-binding site from protein sequence by incorporating spatial and sequence context.
			<b> Scientific Reports</b>, 2016.
			[<a href="./papers/2/PDNAsite_Scientific Reports_2016.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			SPOT-Seq(DNA): Predicting DNA-binding proteins and binding residues by complex structure prediction and application to human proteome.
			<b> PLOS One</b>, 2014.
			[<a href="./papers/2/SPOT-Seq(DNA)_PLOS One_2014.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNABind: A hybrid algorithm for structure-based prediction of DNA-binding residues by combining machine learning- and template-based approaches.
			<b> Proteins</b>, 2013.
			[<a href="./papers/2/DNABind_Proteins_2013.pdf">PDF</a>]
			[<a href="http://mleg.cse.sc.edu/DNABind/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PreDNA: Accurate prediction of DNA-binding sites in proteins by integrating sequence and geometric structure.
			<b> Bioinformatics</b>, 2013.
			[<a href="./papers/2/PreDNA_Bioinformatics_2013.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			COACH: Protein-ligand binding site recognition using complementary binding-specific substructure comparison and sequence profile alignment.
			<b> Bioinformatics</b>, 2013.
			[<a href="./papers/2/COACH_Bioinformatics_2013.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/COACH/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TargetS: Designing template-free predictor for targeting protein-ligand binding sites with classifier ensemble and spatial clustering.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2013.
			[<a href="./papers/2/TargetS_TCBB_2013.pdf">PDF</a>]
			[<a href="http://www.csbio.sjtu.edu.cn/TargetS/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNABR: Sequence-based prediction of DNA-binding residues in proteins with conservation and correlation information.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2012.
			[<a href="./papers/2/DNABR_TCBB_2012.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MetaDBSite: A meta approach to improve protein DNA-binding sites prediction.
			<b> BMC Systems Biology</b>, 2011.
			[<a href="./papers/2/MetaDBSite_BMC Systems Biology_2011.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			BindN+ for accurate prediction of DNA and RNA-binding residues from protein sequence features.
			<b> BMC Systems Biology</b>, 2010.
			[<a href="./papers/2/BindN+_BMC Systems Biology_2010.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DBindR: Prediction of DNA-binding residues in proteins from amino acid sequences using a random forest model with a hybrid feature.
			<b> Bioinformatics</b>, 2009.
			[<a href="./papers/2/DBindR_Bioinformatics_2009.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProteDNA: A sequence-based predictor of sequence-specific DNA-binding residues in transcription factors.
			<b> Nucleic Acids Research</b>, 2009.
			[<a href="./papers/2/ProteDNA_NAR_2009.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			BindN-RF: Prediction of DNA-binding residues from protein sequence information using random forests.
			<b> BMC Bioinformatics</b>, 2009.
			[<a href="./papers/2/BindN-RF_BMC Bioinformatics_2009.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DP-Bind: A web server for sequence-based prediction of DNA-binding residues in DNA-binding proteins.
			<b> Bioinformatics</b>, 2007.
			[<a href="./papers/2/DP-Bind_Bioinformatics_2007.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNABindR: Predicting DNA-binding sites of proteins from amino acid sequence.
			<b> BMC Bioinformatics</b>, 2006.
			[<a href="./papers/2/DNABindR_BMC Bioinformatics_2006.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			BindN: A web-based tool for efficient prediction of DNA and RNA binding sites in amino acid sequences.
			<b> Nucleic Acids Research</b>, 2006.
			[<a href="./papers/2/BindN_NAR_2006.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Pro-DNA: Structure based prediction of binding residues on DNA-binding proteins.
			<b> IEEE Engineering in Medicine and Biology 27th Annual Conference</b>, 2005.
			[<a href="./papers/2/Pro-DNA_IEEE Engineering in Medicine and Biology 27th Annual Conference_2005.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DBS-PSSM: PSSM-based prediction of DNA binding sites in proteins.
			<b> BMC Bioinformatics</b>, 2005.
			[<a href="./papers/2/DBS-PSSM_BMC Bioinformatics_2005.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DBS-Pred: Analysis and prediction of DNA-binding proteins and their binding residues based on composition, sequence and structural information.
			<b> Bioinformatics</b>, 2004.
			[<a href="./papers/2/DBS-Pred_Bioinformatics_2004.pdf">PDF</a>]
		</font>
	</p>
</li>
		</ul>
		</td>
		</tr>
	</table>
</div>
